pixl_imaging/src/pixl_imaging/_processing.py

```diff
@@ -371,28 +334,34 @@ async def _get_missing_instances(
     """
     # First get all SOPInstanceUIDs for the study that are in Orthanc Raw
     orthanc_raw_sop_instance_uids = []
-    for series_id in resource["Series"]:
-        series = await orthanc_raw.query_local_series(series_id)
-        for instance_id in series["Instances"]:
-            instance = await orthanc_raw.query_local_instance(instance_id)
-            orthanc_raw_sop_instance_uids.append(instance["MainDicomTags"]["SOPInstanceUID"])
+    for resource in resources:
+        study_instances = await orthanc_raw.get_local_study_instances(study_id=resource["ID"])
+        orthanc_raw_sop_instance_uids.extend(
+            [instance["MainDicomTags"]["SOPInstanceUID"] for instance in study_instances]
+        )
```

Comment:
chatted about it, but worth documenting in our high-level docs that if we have multiple studies with the same mrn and accesion, then we will merge this in anonymisation